src/main/webapp/registration.html

```diff
@@ -13,18 +14,39 @@
 <!-- this form does not protect against null right now, just is for testing -->
 
 <body>
-	<div id="auto-org-filling-script">
-    <form action="/add-organization" method="POST">
-      <input id="name" type="text" name="org-name" placeholder="Organization Name">
-      <input id="email" type="text" name="email" placeholder="Primary Email">
-      <input id="address" type="text" name="address" placeholder="Street address">
-      <input id = "phone" type = "text" pattern = "[0-9]{10}" name = "phone-number" placeholder="phone number">
-      <input id = "hour-open" type="number" min="0" max="23" name = "hour-open" placeholder="hour you open (0-23)">
-      <input id = "hour-closed" type="number" min="0" max="23" name = "hour-closed" placeholder="hour you close (0-23)">
-      <input id = "urllink" type = "text" name = "url-link" placeholder="website link">
-      <textarea name="description" maxlength="180" placeholder="Org description"></textarea>
-      <input type="submit">
-		</form>
-  </div>
+	<div id="registration-content">
+		<div id="site-title">Givr</div>
+		<div id="registration-form-header">Register An Organization</div>
+		<div id="registration-form-area">
+			<form action="/add-organization" method="POST">
+				<label for = "registration-name" id = "registration-entry-label">Organization Name: </label>
```

Comment:
Did you update AddOrganizationServlet to have all of these new field id's for this form?

Reply:
AddOrganizationServlet actually uses the names and not the ids when it reads in an input request so it still works the name